docs/using_requests_locally.md

```diff
@@ -0,0 +1,104 @@
+# Using Cachito requests locally
+
+The typical mode of interaction with the output of a Cachito request is through
+[OSBS][osbs-cachito]. Installing the content from a finished request without the help of
+OSBS may seem like a daunting task, but is in fact not that difficult. Doing so may be
+especially useful when trying to debug a build failure without going through the entire
+process again.
+
+## Get the relevant files
+
+A Cachito request has three main parts: the **archive**, the **environment variables**
+and the **configuration files**. Getting these manually is a bit of a pain, which is
+why we use the [cachito-download.sh](../bin/cachito-download.sh) script to do it. You
+will need `jq` and the typical unix utils to run this script.
+
+```shell
+cachito-download.sh https://cachito.example.org/api/v1/requests/1 /tmp/cachito-1
+```
+
+## Use the right environment
+
+If you are debugging a build, make sure to match the target environment as closely as
+possible. For example, if you are building a container based on `python:3.9`, then you
+may want to try out the build it that base image.
```

Comment:
@chmeliik Could you please clarify/rewrite the last part of the sentence? 

 `build it that base image`

Reply:
Oh, that's just a typo :smile: 
Does it make more sense now?

Reply:
Sure! Small English words can bring no sense sometimes :D